internal/pkg/manifest/worker_svc.go

```diff
@@ -0,0 +1,210 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package manifest
+
+import (
+	"errors"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/copilot-cli/internal/pkg/template"
+	"github.com/imdario/mergo"
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	workerSvcManifestPath = "workloads/services/worker/manifest.yml"
+)
+
+var (
+	errUnmarshalFifo = errors.New(`cannot unmarshal FIFO field into bool or map`)
```

Comment:
Maybe should be `FIFO` same for below
```suggestion
	errUnmarshalFIFO = errors.New(`cannot unmarshal FIFO field into bool or map`)
```
see https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Adjusted all instances of Fifo -> FIFO.